### PR TITLE
Add user id to the information returned on login

### PR DESCRIPTION
**Pull request by ginold**

### CODE REVIEW

File: c2corg_api/views/user.py [hunk 1]
```diff
@@ -372,6 +372,7 @@ def get(self):
             'name': user.name,
             'forum_username': user.forum_username,
             'id': user.id
+
```
Comment (gberaudo) [inline c2corg_api/views/user.py]: The id line and the blank line should be removed.

Comment (tsauerwein) [inline c2corg_api/views/user.py]: Why also the id line?

Comment (ginold) [inline c2corg_api/views/user.py]: it only appears in the preview, there is no blank line.

Comment (ginold) [inline c2corg_api/views/user.py]: or at least not in my file oO

Comment (gberaudo) [inline c2corg_api/views/user.py]: @tsauerwein, user id should be get on log in, not when editing the user account.

Comment (tsauerwein) [inline c2corg_api/views/user.py]: @gberaudo Ok, it was not clear for me when this view is used.

Comment (tsauerwein) [inline c2corg_api/views/user.py]: @ginold The version that you want to merge contains a blank line. Make sure that you have pushed the latest version.

**Pull request merged**